tests/test/existential_types.rs

```diff
@@ -14,12 +14,6 @@ fn dyn_Clone_is_Clone() {
         } yields {
             "Unique; substitution []"
         }
-
-        goal {
-            impl Clone: Clone
```

Comment:
Should these tests be elsewhere? (Or just commented out?) Or are `impl Trait`s just going to be in lowering now? Presumably, we *do* want these tests in some form?

Reply:
I think they should just be removed for now. We'll have to figure out how to test impl Trait when we grow support for it